dusty/source.py

```diff
@@ -100,7 +100,10 @@ def ensure_local_repo(self):
         if not os.path.exists(repo_path_parent):
             os.makedirs(repo_path_parent)
         with git_error_handling():
-            managed_repo = git.Repo.clone_from('ssh://{}@{}'.format(constants.GIT_USER, self.remote_path), self.managed_path)
+            if self.is_local_repo:
+                git.Repo.clone_from('file:///{}'.format(self.remote_path), self.managed_path)
```

Comment:
seems like a weird pattern.  but i guess it would work.  It would continually pull from a local git repo to the dusty managed version.  Interesting way to do local dev without overrides needed